bool mode
float64 a
float64 b
float64 c
float64 d
float64 e
float64 f
string[] MDIS
string[] paramValue
---
int32 res